PyTorchEx1.ipynb

```diff
@@ -0,0 +1,478 @@
+{
```

Comment:
<p>Can we do it to a fixed length to keep the example minimal , so something like , maybe something like  256 ? </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/rapidsai/rapids-examples/pull/32/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='19'/>